src/devices/ovs/nm-device-ovs-bridge.c

```diff
@@ -134,6 +134,9 @@ nm_device_ovs_bridge_init (NMDeviceOvsBridge *self)
 static const NMDBusInterfaceInfoExtended interface_info_device_ovs_bridge = {
 	.parent = NM_DEFINE_GDBUS_INTERFACE_INFO_INIT (
 		NM_DBUS_INTERFACE_DEVICE_OVS_BRIDGE,
+		.properties = NM_DEFINE_GDBUS_PROPERTY_INFOS (
+			NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE_L ("Slaves", "ao", NM_DEVICE_SLAVES),
```

Comment:
new code should not use `NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE_L`. Instead, use `NM_DEFINE_DBUS_PROPERTY_INFO_EXTENDED_READABLE`. The "L" stands for "legacy".